crates/sui-types/src/transaction.rs

```diff
@@ -2259,17 +2273,6 @@ impl SenderSignedData {
 
         Ok(())
     }
-
-    pub fn verify_max_epoch_for_all_sigs(
```

Comment:
Could you point out why it is ok to remove this function?

Reply:
all authentication goes through this path now: https://github.com/MystenLabs/sui/blob/b89eb622bbbf9d1c64d6b2a8855548a6bc2f78f2/crates/sui-types/src/signature.rs#L109